src/librustdoc/html/render/context.rs

```diff
@@ -206,7 +206,7 @@ impl<'tcx> Context<'tcx> {
         if !is_module {
             title.push_str(it.name.unwrap().as_str());
         }
-        if !it.is_primitive() && !it.is_keyword() {
+        if !it.is_primitive() && !it.is_keyword() && !it.is_attribute() {
```

Comment:
It might be worth adding a `clean::Item::is_std_synthetic` method for all 3 of these cases (name heavily subject to bikeshedding).

Reply:
I thought about the same but wasn't sure it was worth it. But since I'm not the only one thinking that, adding it.